src/components/craft/craft.tsx

```
@@ -9,19 +9,24 @@ interface CraftProps {
 
 export function Craft({ scenario }: CraftProps) {
   const { destAirportInfo, craft } = scenario;
+  const clearanceLimit = destAirportInfo?.name ?? craft?.clearanceLimit;
+  const departure = craft?.frequency ?? "offline";
```

ESLint

[error] 'departure' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)